This fixes an error where the path '/' is attempted to be converted from a
relative to absolute path on windows, resulting in a ValueError.

Note that this patch stops setup.py ... bdist_wheel --universal from being
called, so this patch may need to be reconsidered if we want this to be made a
noarch: python package for any reason.
Index: python-coloredlogs/setup.py
===================================================================
--- python-coloredlogs.orig/setup.py
+++ python-coloredlogs/setup.py
@@ -80,8 +80,7 @@ def find_pth_directory():
     ``*.pth`` files are installed in the ``lib/pythonX.Y/site-packages``
     directory without hard coding its location.
     """
-    return ('/' if 'bdist_wheel' in sys.argv
-            else os.path.relpath(distutils.sysconfig.get_python_lib(), sys.prefix))
+    return (os.path.relpath(distutils.sysconfig.get_python_lib(), sys.prefix))
 
 
 setup(name='coloredlogs',
